subprojects/language-groovy/src/testFixtures/groovy/org/gradle/testing/fixture/GroovyCoverage.groovy

```diff
@@ -46,17 +46,17 @@ class GroovyCoverage {
 
     static {
         SUPPORTED_BY_JDK = groovyVersionsSupportedByJdk(JavaVersion.current())
-        SUPPORTS_GROOVYDOC = versionsAbove(SUPPORTED_BY_JDK, "1.6.9")
+        SUPPORTS_GROOVYDOC = versionsNotLessThan(SUPPORTED_BY_JDK, "1.6.9")
         // Indy compilation doesn't work in 2.2.2 and before
-        SUPPORTS_INDY = versionsAbove(SUPPORTED_BY_JDK, "2.3.0")
-        SUPPORTS_TIMESTAMP = versionsAbove(SUPPORTED_BY_JDK, "2.4.6")
-        SUPPORTS_PARAMETERS = versionsAbove(SUPPORTED_BY_JDK, "2.5.0")
-        SUPPORTS_DISABLING_AST_TRANSFORMATIONS = versionsAbove(SUPPORTED_BY_JDK, "2.0.0")
-        SINCE_3_0 = versionsAbove(SUPPORTED_BY_JDK, "3.0.0")
+        SUPPORTS_INDY = versionsNotLessThan(SUPPORTED_BY_JDK, "2.3.0")
```

Comment:
Have you verified that all of these versions are still correct given the change in behavior between `versionsAbove` and `versionsNotLessThan`?

Reply:
My changes do not change the semantics. `versionsAbove` was an inclusive version.

Reply:
Shouldn't `versionsNotLessThan ` and `versionsNotGreaterThan` be calling `filterVersionsStrictly` in order to not change behavior, since `filterVersions` behavior has changed